### PR TITLE
feat(TFD-9887): active list element

### PR DESCRIPTION
**Pull request by silvanocostanzo**
**What is the problem this PR is trying to solve?**
The active tab, which is shown as first element in the dropdown, should not appear again in the list when the dropdown menu is unfolded. (i.e. Configuration appears as first element, then when the dropdown menu is unfolded only Connection, Info, Dataset should be shown) 
connection_dropdown_after
Related PR in data-streams:
https://github.com/Talend/data-streams/pull/1844

Jira:
https://jira.talendforge.org/browse/TFD-9887

**What is the chosen solution to this problem?**
The selected element only appears in the title and not in the list

**Please check if the PR fulfills these requirements**

* [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
* [x] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/components/src/Actions/ActionDropdown/ActionDropdown.component.js [hunk 1]
```diff
@@ -59,32 +59,33 @@ InjectDropdownMenuItem.propTypes = {
...
+	const activeListElement = index === activeKey;
 	return (
 		<Renderers.MenuItem
 			{...item}
 			key={index}
 			eventKey={item}
 			onClick={wrapOnClick(item)}
 			title={item.title || item.label}
-			className={classNames(theme['tc-dropdown-item'], 'tc-dropdown-item')}
+			className={classNames(theme['tc-dropdown-item'], 'tc-dropdown-item', {activeListElement})}
```
Comment (tlnd-mhuchet) [inline packages/components/src/Actions/ActionDropdown/ActionDropdown.component.js:75]: I think there is some css class missing along the activeListElement boolean?

File: packages/components/src/Actions/ActionDropdown/ActionDropdown.component.js [hunk 2]
```diff
@@ -227,7 +228,7 @@ class ActionDropdown extends React.Component {
 				aria-label={tooltipLabel || label}
 				{...omit(rest, 'tReady')}
 				onToggle={this.onToggle}
-				ref={ref => (this.ref = ref)}
+				ref={(ref) => (this.ref = ref)}
```
Comment (tlnd-mhuchet) [inline packages/components/src/Actions/ActionDropdown/ActionDropdown.component.js:231]: No need to add parenthesis here :)

File: output/components.eslint.txt [hunk 3]
```diff
@@ -2,6 +2,9 @@
 
 The react/require-extension rule is deprecated. Please use the import/extensions rule from eslint-plugin-import instead.
 
+/home/travis/build/Talend/ui/packages/components/src/Actions/ActionDropdown/ActionDropdown.component.js
+  193:4  error  'activeKey' is missing in props validation  react/prop-types
```
Comment (jsomsanith-tlnd) [inline output/components.eslint.txt:6]: Can you fix this one please ?

Comment (silvanocostanzo) [reply output/components.eslint.txt:6]: yes, thanks, i missed this warning

**Comment by silvanocostanzo** [comment]
> I think we'll have to know what we want to do on the UX side as we may don't really want some specific behaviour for this component in TPD.
> Let's check the point with the UX first

for the moment we put it on hold, we need further discussions

**Comment by silvanocostanzo** [comment]
no more needed

**Pull request closed without merging**